odk1-src/central-forms.rst

```diff
@@ -39,34 +44,53 @@ You can either click on the :guilabel:`choose one` button to browse for your :fi
    - You may see a message that reads **A resource already exists with xmlFormId value(s) of xyz.** If you do, there already exists a form within this project with the same unique designation. If you are using XLSForm, try changing the name of the file or the ``form_id`` in the settings sheet. If you designed the form by hand, please check the ``id="…"`` attribute immediately inside the ``<instance>`` tag.
    - You may see a message that says **A form previously existed which had the same formId and version as the one you are attempting to create now. To prevent confusion, please change one or both and try creating the form again.** This means there once was a form within this project that has since been deleted that has exactly the same formId (see the previous bullet point) *and* version designation as the one you are now trying to upload. Central won't accept the new form, because this conflict could cause confusion with mobile devices that still have the old form sitting around. To upload this form, change either the formId (again, see the previous bullet point) or `update the version <https://opendatakit.github.io/xforms-spec/#primary-instance>`_ and try again.
 
-Once the form is successfully uploaded, you will be taken to the Form Overview page.
+Once the form is successfully uploaded, you will be taken to the Form Draft page. It will not be accessible to data collection clients until you publish the Draft, which we will cover in the following section.
+
+.. _central-forms-draft:
+
+Working With Form Drafts
```

Comment:
I'd call this "Form Drafts" to match the style of the rest of the headers.

Reply:
i did this, but later coming back to this file i actually think most of the headers use an action phrase so i changed the passive ones to action ones.

Reply:
Just to check, does this affect the links in Frontend at all (https://github.com/getodk/central-frontend/pull/308)? For example, should I change .../central-forms/#form-drafts to .../central-forms/#understanding-form-drafts?

Reply:
oh shoot. i forgot that the url token is based on the human name rather than the assigned ID. ughhhhhhh rst is so confusing.

i actually wanted to eliminate that header altogether but i moved it instead because i thought it would save you an edit. i think since i've screwed you over i'll remove that header and you can change instances of form-draft-tabs and form-drafts to working-with-form-drafts.

sorry

Reply:
It's definitely a super easy change for Frontend! I'll go ahead and update those links. :+1: